dev/com.ibm.ws.jaxws.tools/src/com/ibm/ws/jaxws/tools/WsToolsUtils.java

```diff
@@ -19,9 +19,9 @@
 /**
  * Collected methods used by WsGen and WsImport in a utility class*
  */
-class WsToolsUtils {
+public class WsToolsUtils {
```

Comment:
Had to make this public since I renamed the package for the wsgen and wsimport wrappers in xmlWS-3.0